ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -168,3 +168,39 @@ keys:
                           - "mbytes"
                           - "packets"
                         default: "bytes"
+      copp_system_policy:
+        type: dict
+        description: Control-plane policy configuration.
+        keys:
+          enable:
```

Comment:
We can remove enable and check copp_system_policy in j2 directly

Reply:
Fixed.